avm/utilities/pipelines/staticValidation/compliance/helper/helper.psm1

```diff
@@ -221,7 +221,10 @@ function Resolve-ReadMeParameterList {
 
     $parameterSet = @{}
 
-    if (-not $Properties) {
+    if (-not $Properties -and -not $TemplateFileContent.parameters) {
```

Comment:
just checking if this is related to utl support

Reply:
It was in that the utility module that Alex F wants to contribute does not have parameters. And this logic effectively produced a nullPointerException in that case.
Fixed it in the same way somewhere else 😏